<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyWindow</class>
 <widget class="QMainWindow" name="MyWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>720</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>900</width>
    <height>720</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>900</width>
    <height>720</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Transformacje 3D</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>cube.ico</normaloff>cube.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QFrame" name="rysujFrame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>600</width>
      <height>600</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="exitButton">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>660</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Zakończ program</string>
    </property>
   </widget>
   <widget class="QPushButton" name="resetButton">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>620</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Resetuj wartości</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="translationBox">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>80</y>
      <width>251</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>Translation</string>
    </property>
    <widget class="QSlider" name="translationXSlider">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>26</x>
       <y>30</y>
       <width>200</width>
       <height>16</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-256</number>
     </property>
     <property name="maximum">
      <number>256</number>
     </property>
     <property name="singleStep">
      <number>1</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QSlider" name="translationYSlider">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>26</x>
       <y>60</y>
       <width>200</width>
       <height>16</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-256</number>
     </property>
     <property name="maximum">
      <number>256</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QSlider" name="translationZSlider">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>26</x>
       <y>90</y>
       <width>200</width>
       <height>16</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-256</number>
     </property>
     <property name="maximum">
      <number>256</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="translationZButton">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>90</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>X</string>
     </property>
    </widget>
    <widget class="QPushButton" name="translationYButton">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>60</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>X</string>
     </property>
    </widget>
    <widget class="QPushButton" name="translationXButton">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>30</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>X</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="scalingBox">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>340</y>
      <width>251</width>
      <height>131</height>
     </rect>
    </property>
    <property name="title">
     <string>Scaling</string>
    </property>
    <widget class="QSlider" name="scalingXSlider">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>26</x>
       <y>30</y>
       <width>200</width>
       <height>16</height>
      </rect>
     </property>
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>200</number>
     </property>
     <property name="value">
      <number>100</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QSlider" name="scalingYSlider">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>26</x>
       <y>60</y>
       <width>200</width>
       <height>16</height>
      </rect>
     </property>
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>200</number>
     </property>
     <property name="value">
      <number>100</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QCheckBox" name="scalingCheckBox">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>108</y>
       <width>241</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Zachowaj proporcje sześcianu foremnego</string>
     </property>
    </widget>
    <widget class="QSlider" name="scalingAllSlider">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>26</x>
       <y>60</y>
       <width>200</width>
       <height>16</height>
      </rect>
     </property>
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>200</number>
     </property>
     <property name="value">
      <number>100</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QSlider" name="scalingZSlider">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>26</x>
       <y>90</y>
       <width>200</width>
       <height>16</height>
      </rect>
     </property>
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>200</number>
     </property>
     <property name="value">
      <number>99</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="scalingXButton">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>30</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>X</string>
     </property>
    </widget>
    <widget class="QPushButton" name="scalingYButton">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>60</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>X</string>
     </property>
    </widget>
    <widget class="QPushButton" name="scalingZButton">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>90</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>X</string>
     </property>
    </widget>
    <widget class="QPushButton" name="scalingAllButton">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>60</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>X</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="rotationBox">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>210</y>
      <width>251</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>Rotation</string>
    </property>
    <widget class="QSlider" name="rotationXSlider">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>26</x>
       <y>30</y>
       <width>200</width>
       <height>16</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-180</number>
     </property>
     <property name="maximum">
      <number>180</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QSlider" name="rotationYSlider">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>26</x>
       <y>60</y>
       <width>200</width>
       <height>16</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-180</number>
     </property>
     <property name="maximum">
      <number>180</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QSlider" name="rotationZSlider">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>26</x>
       <y>90</y>
       <width>200</width>
       <height>16</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-180</number>
     </property>
     <property name="maximum">
      <number>180</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="rotationYButton">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>60</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>X</string>
     </property>
    </widget>
    <widget class="QPushButton" name="rotationXButton">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>30</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>X</string>
     </property>
    </widget>
    <widget class="QPushButton" name="rotationZButton">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>90</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>X</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="rzutowanieBox">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>10</y>
      <width>251</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string>&quot;d&quot; parameter</string>
    </property>
    <widget class="QSlider" name="rzutowanieSlider">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>26</x>
       <y>33</y>
       <width>200</width>
       <height>16</height>
      </rect>
     </property>
     <property name="minimum">
      <number>500</number>
     </property>
     <property name="maximum">
      <number>1200</number>
     </property>
     <property name="value">
      <number>500</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="rzutowanieXButton">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>32</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>X</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="shearingBox">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>480</y>
      <width>251</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>Shearing</string>
    </property>
    <widget class="QSlider" name="shearingXSlider">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>26</x>
       <y>30</y>
       <width>200</width>
       <height>16</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-100</number>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QSlider" name="shearingYSlider">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>26</x>
       <y>60</y>
       <width>200</width>
       <height>16</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-100</number>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QSlider" name="shearingZSlider">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>26</x>
       <y>90</y>
       <width>200</width>
       <height>16</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-100</number>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="shearingYButton">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>60</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>X</string>
     </property>
    </widget>
    <widget class="QPushButton" name="shearingZButton">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>90</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>X</string>
     </property>
    </widget>
    <widget class="QPushButton" name="shearingXButton">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>30</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>X</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="colorCButton">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>620</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Kolor Krawędzi Bocznych</string>
    </property>
   </widget>
   <widget class="QPushButton" name="colorBButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>660</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Kolor Podstawy Dolnej</string>
    </property>
   </widget>
   <widget class="QPushButton" name="colorAButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>620</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Kolor Podstawy Górnej</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="gruboscBox">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>635</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QSlider" name="gruboscSlider">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>9</y>
       <width>169</width>
       <height>16</height>
      </rect>
     </property>
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>6</number>
     </property>
     <property name="value">
      <number>2</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="gruboscLabel">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>22</y>
       <width>130</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Grubość wierzchołków</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="colorButton">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>660</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Kolor Wierzchołków</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
